src/commands/executeFunction.ts

```diff
@@ -42,5 +82,6 @@ export async function executeFunction(context: IActionContext, node?: FunctionTr
         }
     });
 
-    window.showInformationMessage(localize('executed', 'Executed function "{0}"', name));
+    const message: string = response ? localize('executedWithResponse', 'Executed function "{0}". Response: "{1}"', node.name, response) : localize('executed', 'Executed function "{0}"', node.name);
```

Comment:
Do the timer triggers have responses?  I noticed that you didn't output it before.

Reply:
They don't for the default template. Technically I think users can change the code and add a response for any function type